support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -19,7 +19,7 @@ Next:
 Next:
   Run: spectre.Pipelines.Bbh.EccentricityControl:eccentricity_control
   With:
-    h5_files: ./BbhReductions.h5
+    h5_files: ../Segment_*/BbhReductions.h5
```

Comment:
The Test_Inspiral.py currently fails because you have to adjust this field there as well.